plugins/chat/assets/javascripts/discourse/components/chat-message-reaction.hbs

```diff
@@ -8,11 +8,14 @@
       (if this.isActive "-active")
     }}
     data-emoji-name={{@reaction.emoji}}
-    data-tippy-content={{this.popoverContent}}
     title={{this.emojiString}}
-    {{did-insert this.setup}}
     {{will-destroy this.teardown}}
-    {{did-update this.refreshTooltip this.popoverContent}}
+    {{on "click" this.handleClick passive=true}}
```

Comment:
Just dropping the comment here because it's this component, needs an extra space after the text:

![image](https://github.com/discourse/discourse/assets/920448/15c270bd-37f9-41cb-b61f-547a38f8a7ca)

Reply:
yes I saw this right, I need to fix it :+1: